HPXMLtoOpenStudio/resources/hvac_sizing.rb

```diff
@@ -1651,180 +1650,170 @@ def self.apply_hvac_installation_quality(hvac_sizing_values, weather, hvac)
+    # Cooling
+    if [HPXML::HVACTypeHeatPumpAirToAir,
+        HPXML::HVACTypeCentralAirConditioner,
+        HPXML::HVACTypeHeatPumpMiniSplit,
+        HPXML::HVACTypeMiniSplitAirConditioner,
+        HPXML::HVACTypeHeatPumpGroundToAir].include?(hvac.CoolType) && hvac.CoolingLoadFraction > 0
 
-        cap_htg_ratio = 1 / (totalCap_CurveValue_d / totalCap_CurveValue)
-        if cap_htg_ratio > 1
-          hvac_sizing_values.Heat_Capacity *= cap_htg_ratio
-          hvac_sizing_values.Heat_Airflow = calc_airflow_rate(hvac_sizing_values.Heat_Capacity, (hvac.SupplyAirTemp - @heat_setpoint))
-        end
-      end
-    else
-      f_ch = hvac.ChargeDefectRatio.round(3)
-
-      # Cooling
-      if [HPXML::HVACTypeHeatPumpAirToAir,
-          HPXML::HVACTypeCentralAirConditioner,
-          HPXML::HVACTypeHeatPumpMiniSplit,
-          HPXML::HVACTypeMiniSplitAirConditioner].include?(hvac.CoolType) && hvac.CoolingLoadFraction > 0
-        cool_airflow_rated_defect_ratio = []
-        cool_airflow_rated_ratio = []
-        cool_cfm_m3s = UnitConversions.convert(hvac_sizing_values.Cool_Airflow, 'cfm', 'm^3/s')
+      cool_airflow_rated_defect_ratio = []
+      cool_airflow_rated_ratio = []
+      cool_cfm_m3s = UnitConversions.convert(hvac_sizing_values.Cool_Airflow, 'cfm', 'm^3/s')
+      if not hvac.CoolType == HPXML::HVACTypeHeatPumpGroundToAir
         for speed in 0..(hvac.NumSpeedsCooling - 1)
           cool_airflow_rated_ratio << cool_cfm_m3s / HVAC.calc_rated_airflow(hvac_sizing_values.Cool_Capacity, hvac.RatedCFMperTonCooling[speed], hvac.CapacityRatioCooling[speed])
           cool_airflow_rated_defect_ratio << cool_cfm_m3s * (1 + hvac.AirflowDefectRatioCooling) / HVAC.calc_rated_airflow(hvac_sizing_values.Cool_Capacity, hvac.RatedCFMperTonCooling[speed], hvac.CapacityRatioCooling[speed])
         end
-        if not cool_airflow_rated_defect_ratio.empty?
-          cap_clg_ratios = []
-          for speed in 0..(hvac.NumSpeedsCooling - 1)
-            # NOTE: heat pump (cooling) curves don't exhibit expected trends at extreme faults;
+      else
+        cool_airflow_rated_ratio = [1.0] # actual air flow is equal to rated (before applying defect ratio) in current methodology
+        cool_airflow_rated_defect_ratio = [1 + hvac.AirflowDefectRatioCooling]
+      end
+      if not cool_airflow_rated_defect_ratio.empty?
+        cap_clg_ratios = []
+        for speed in 0..(hvac.NumSpeedsCooling - 1)
+          # NOTE: heat pump (cooling) curves don't exhibit expected trends at extreme faults;
+          if not hvac.CoolType == HPXML::HVACTypeHeatPumpGroundToAir
             a1_AF_Qgr_c = hvac.COOL_CAP_FFLOW_SPEC[speed][0]
             a2_AF_Qgr_c = hvac.COOL_CAP_FFLOW_SPEC[speed][1]
             a3_AF_Qgr_c = hvac.COOL_CAP_FFLOW_SPEC[speed][2]
+          else
+            # Fixme: Please review these coefficients. Relationship independent of other variables to mimic fFF curves.
```

Comment:
Are we comfortable enough to remove this FIXME? We can't merge the code w/ a FIXME still remaining in the code.